docs/BUILD.md

```diff
@@ -8,28 +9,36 @@ s2n-tls supports and tests on **x86** and **arm** architectures.
 
 ### System requirements
 
-* 20GB RAM available
+* 20GB RAM available, recommended.
+
+### Supported OS and Distributions 
+
+We support current and LTS versions of Ubuntu and macOS distributions.
 
-### Supported OS and Distributions: 
+#### Linux
 
-| OS     | Distros |      |        |
-|--------|---------|------|--------|
-| Redhat | Fedora  | AL2  |
-| debian | ubuntu  |
-| *BSD   | free    | open | darwin |
+We support current and standard security LTS Ubuntu distributions.
 
-s2n-tls does not support [Windows](https://github.com/aws/s2n-tls/issues/497).
+We do not support Linux distributions that are compatible with what we test. For your convenience, these are shown as "untested, unsupported" in our support matrix.
 
-<!-- We may want to move in this direction:
+| Distro | Support | Supported Versions |
```

Comment:
@dougch This could use more refinement, but what do you think of it as a direction?